src/bp/core/services/hook/hook-service.ts

```diff
@@ -40,6 +40,8 @@ export namespace Hooks {
     }
   }
 
+  export class EventHook extends BaseHook {}
```

Comment:
I'm just curious of why we do this?

Reply:
Wanted to only check for specific type of events, but I realized that I just need to check if there's an argument "event", so I've removed it